packages/cli/src/lib/Cli.ts

```diff
@@ -89,14 +88,14 @@ export class Cli {
         return await merge(parsedArgs);
       case 'fingerprint':
         return await fingerprint(parsedArgs);
-      // case 'play':
-      //   return await play(parsedArgs as unknown as PlayArgs);
-      // case 'playPublish':
-      //   return await play(parsedArgs as unknown as PlayArgs, 'publish');
-      // case 'playVersionCheck':
-      //     return await play(parsedArgs as unknown as PlayArgs, 'versionCheck');
-      // case 'playRetain':
-      //     return await play(parsedArgs as unknown as PlayArgs, 'retain');
+      case 'play':
+        return await play(parsedArgs as unknown as PlayArgs);
```

Comment:
Hi Jakub, did you test all 4 of these new commands? I doubt that the `play` command would work, because the `play()` function requires the second parameter `command`. I am not sure what the original author meant by this, but I suggest deleting the two lines above.

Reply:
hey, you're right - I filed this PR rather lazily.
Check out new commit: I've move play commands into single namespace (so, all of the `publish`, `retain`, `versionCheck` commands are subcommands of `bubblewrap play`) and cleanup the implementation a bit - now it resembles other parts of the codebase more (mainly fingerprint/build commands).

I've also tested once again `publish` and `versionCheck` - they are working. Not sure about the retain - that's why I'm leaving it marked as "experimental" in the docs and in the implementation (there is an additional prompt, so the needs to confirm when they want to use an experimental feature).

What do you think?